test/system/static.test.ts

```diff
@@ -633,3 +633,71 @@ test("static scanning NGINX with dockerfile analysis matches expected values", a
     "found gnupg1 in dockerfile packages",
   );
 });
+
+test("inspect registry.suse.com/suse/sle15:15.1", (t) => {
```

Comment:
Nit: prefer using async and then awaiting later in the test, instead of using `.then()`